packages/bitcoin-ledger-provider/lib/BitcoinLedgerProvider.ts

```diff
@@ -117,19 +145,20 @@ export default class BitcoinLedgerProvider extends BitcoinWalletProvider(LedgerP
         .toString('hex')
       const isSegwit = [bitcoin.AddressType.BECH32, bitcoin.AddressType.P2SH_SEGWIT].includes(this._addressType)
       const changeAddress = await this.findAddress(
-        psbt.txOutputs.map((output) => output.address),
+        psbt.txOutputs.map((output) => addrToBitcoinJS(output.address, this._network)), // todotest or not???
```

Comment:
I'm not sure if `addrToBitcoinJS` is needed here. Please run integration tests twice, once with and once without.

Reply:
With and without got me to the final step of sending the redeem tx (fails with same error)